### PR TITLE
Move nopImage and entrypointImage from pkg/… package to cmd/controller

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This is part of a set of changes to make sure we don't define CLI
flags in our `pkg/…` packages… so that importing packages from there
do not pollute the cli flags.

This is part of #1309 — submitting this one *alone* to gather early feedback and make sure this is the way to go (aka I am not a fan of the map in this change so far)

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) to build and release this image.

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md) are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes) must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS) and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes) must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS), and they must first be added [in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

### CODE REVIEW

File: cmd/controller/main.go [hunk 1]
```diff
@@ -28,9 +30,20 @@ const (
...
 
+var (
+	entrypointImage = flag.String("entrypoint-image", "override-with-entrypoint:latest",
+		"The container image containing our entrypoint binary.")
+	nopImage = flag.String("nop-image", "override-with-nop:latest",
+		"The container image used to kill sidecars")
+)
+
 func main() {
+	images := map[string]string{
```
Comment (ImJasonH) [inline cmd/controller/main.go]: Maybe an `Images` struct with named fields? Having a `map` makes it seem like people should be able to add any image they want, and invites typos.

**Comment by vdemeester** [comment]
Updated :angel: 
/cc @ImJasonH @bobcatfish @sbwsg

**Comment by vdemeester** [comment]
/hold cancel
I'll update the rest of the flags in a follow-up :wink:

**Pull request merged**